src/api/entities/confidential/__tests__/ConfidentialTransaction/index.ts

```diff
@@ -149,6 +152,85 @@ describe('ConfidentialTransaction class', () => {
     });
   });
 
+  describe('method: onStatusChange', () => {
+    let bigNumberToU64Spy: jest.SpyInstance;
+    let instructionStatusesMock: jest.Mock;
+    const rawId = dsMockUtils.createMockU64(new BigNumber(1));
+
+    afterAll(() => {
+      jest.restoreAllMocks();
+    });
+
+    beforeAll(() => {
+      bigNumberToU64Spy = jest.spyOn(utilsConversionModule, 'bigNumberToU64');
+    });
+
+    beforeEach(() => {
+      const owner = 'someDid';
+      entityMockUtils.configureMocks({ identityOptions: { did: owner } });
+      when(bigNumberToU64Spy).calledWith(id, context).mockReturnValue(rawId);
+
+      instructionStatusesMock = dsMockUtils.createQueryMock(
```

Comment:
```suggestion
      transactionStatusesMock = dsMockUtils.createQueryMock(
```
nit -> feels like got left over from copy paste

Reply:
😅